greengrass_defender_agent/config.py

```diff
@@ -10,13 +10,21 @@
 
 # Set all the constants
 QOS_TYPE = QOS.AT_MOST_ONCE
+IPC_CONNECT_TIMEOUT = 120
```

Comment:
I'm creating a new timeout constant here because the original `TIMEOUT` was applied universally to every async call in this agent. Doesn't make much sense to increase all of them to a extreme long waiting time.